### ISSUE TITLE
Organisations SVGs need explicit width or height

### ISSUE THREAD

**Issue by lmammino** [fastify/website-metalsmith#91]
Nucleode logo is there but it displays with 0x0 pixels. I will fix this specific case manually and I will try to identify a way to avoid this from happening again before merging PRs.

CC: @mcollina @fox1t @cemremengu

**Comment by cemremengu** [comment]
My apologies @lmammino, I will make sure to ask for your review first next time

**Comment by lmammino** [comment]
No problem at all @cemremengu, I probably wouldn't have spotted that either 😉

**Comment by fox1t** [comment]
I am really sorry. This logo was made by our designer @StefanoMiC, maybe he can help about identifying a way to avoid this issue.

**Comment by lmammino** [comment]
@fox1t, I believe it was the compressor I recommended in the guide to remove those attributes. So I should be the one to take all the blame (even though I am not trying to blame anyone! 😛).

It's good to have a new logo and an improved process for logo submissions. So thanks again for your support.

If you and/or @StefanoMiC, want to have a look at the current process check out the changes in #93

**Comment by fox1t** [comment]
OK I am going to be killed by @StefanoMiC because I used the compressor on our logo. :(

**Comment by StefanoMiC** [comment]
Hi @lmammino, I've managed to look into the issue and noticed a discrepancy between the file @fox1t pushed and the SVG that appeared on the website, the structure was substantially different. I've manually fixed the code in which some shapes got misplaced in groups and some colors mixed up, not sure why though.. I'm also not sure if an automated minifier is the best choice in case there are any gradients applied, that's maybe worth further testing.  
Thank you for your help!

Hope  [#94](https://github.com/fastify/website/pull/94) resolves the issue.
